common/app/views/support/EmailHelpers.scala

```diff
@@ -24,6 +25,12 @@ object EmailHelpers {
     case p: PressedPage => p.frontProperties.onPageDescription
   }
 
+  def classesForCard(card: ContentCard, withImage: Boolean = false): Seq[String] = {
+    Seq(toneClassFromStyle(card.cardStyle)) ++
+      (if(withImage) Seq("fc--large") else Nil) ++
+      (if(card.branding.isDefined) Seq("tone-branded") else Nil)
```

Comment:
Could the card ever have sponsored branding instead of paid branding? eg /cities is branded but doesn't have paid branding. Not sure if that's relevant here or not.